src/commands/request.js

```diff
@@ -48,8 +48,12 @@ export default class RequestCommand extends GrouperCommand {
         }
 
         response
-            .addField(`**${sTag}**`, `\`\`\`${grouper.joinArgAfter(1)}\`\`\``);
+            .setTitle("TAG REQUEST")
+            .setColor("GREEN")
+            .addField(`**${sTag}**`, `\`\`\`${grouper.joinArgAfter(1)}\`\`\``)
+            .setFooter(`Requested by ${grouper.message.author.username}#${grouper.message.author.discriminator} in ${grouper.message.guild.name}`);
 
+        grouper.message.reply("Request successful! ✅")
```

Comment:
A little undecided if we should use simple text reply for this command while we use embed for all other ones, seems inconsistent.